Use correct min MySQL version for axis-order (8.0.1)

MoonE opened the pull request:
There was no version 8.0.10 (`80010`), axis-order was introduced in MySQL 8.0.1
https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html

Please tell me if  you want me to rebase this on a different branch as a bugfix

Signed-off-by: Maximilian Krög <maxi_kroeg@web.de>

williamdes commented:
Hi @MoonE 
Could you rebase this fix onto QA_5_1, force push and then change the PR target using the edit button ?

williamdes commented:
Confirmed by #16088 and my comment https://github.com/phpmyadmin/phpmyadmin/issues/16012#issuecomment-616215760

MoonE committed:
Use correct min MySQL version for axis-order (8.0.1)

There was no version 8.0.10
https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html

Signed-off-by: Maximilian Krög <maxi_kroeg@web.de>

williamdes commented:
Thank you @MoonE for your contribution !
